### PR TITLE
Us024   create authentication system

### PR DESCRIPTION
**Pull request by atracey1**

### CODE REVIEW

File: tests/uiTests/loginTests.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  this.timeout(0);
+  beforeEach(async () => {
+    driver = await new webdriver.Builder().forBrowser('chrome').build();
+  });
+
+  afterEach(async () => {
+    await driver.quit();
+  });
+
+  it('should give appropiate menssage if incorrect userame provided', async () => {
```
Comment (Matthew-Kainos) [inline tests/uiTests/loginTests.js]: just a typo in the message '' should give appropriate message if incorrect username provided.

File: tests/unitTests/login.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      const hashedPw = await bcrypt.hash('secret2', 10);
+      const postStub = sinon.stub(app, 'getPassword');
+      postStub.resolves({ data: hashedPw });
+      const result = await checkIfPasswordValid('abc', password);
+      expect(result).equal(false);
+      postStub.restore();
+    });
+  });
+  describe('initiliseSessionPermissions', async () => {
+    it('Should successfuly initilise admin session', async () => {
```
Comment (Matthew-Kainos) [inline tests/unitTests/login.js]: typo - Initialise

**Review by Matthew-Kainos — APPROVED** [review]

File: tests/uiTests/loginTests.js (hunk 1 above)
Comment (atracey1) [reply tests/uiTests/loginTests.js]: done

File: tests/unitTests/login.js (hunk 2 above)
Comment (atracey1) [reply tests/unitTests/login.js]: done

**Review by GHNelson752 — APPROVED** [review]

**Pull request merged**